source/camera/CameraManager.h

```diff
@@ -86,16 +90,20 @@ class CameraManager
         return mTranslateVectorAccel;
     }
 
-    bool onFrameStarted()
-    { return true; }
-    bool onFrameEnded()
-    { return true; }
+    bool onFrameStarted();
+    bool onFrameEnded();
+
+    CullingManager* mCullingManager;
 
     /*! \brief Sets the camera to a new location while still satisfying the
     * constraints placed on its movement
     */
     void updateCameraFrameTime(const Ogre::Real frameTime);
 
+    void startCreatureCulling();
```

Comment:
why don't you have a stopCreatureCulling if you have it for tiles ? It is not a big problem but is it not a missed function ?